src/core/columns/right.py

```diff
@@ -0,0 +1,22 @@
+from attrs import define
+import streamlit as st
+
+from src.states.gui import GUIState
```

Comment:
Let's keep order of the modules aplhabetically. Lines 4 and 5 should be switched.